### PR TITLE
DDT-764: Refactor EBL to use event-core

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: src/main/java/org/dcsa/ebl/service/impl/BookingServiceImpl.java [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
-@Service
-public class BookingServiceImpl extends ExtendedBaseServiceImpl<BookingRepository, Booking, String> implements BookingService {
-    private final BookingRepository bookingRepository;
-
-    @Override
-    public BookingRepository getRepository() {
-        return bookingRepository;
-    }
-}
+//package org.dcsa.ebl.service.impl;
```
Comment (pimg) [inline src/main/java/org/dcsa/ebl/service/impl/BookingServiceImpl.java:1]: either remove it or if it shouldn't be removed the BookingRepository needs to be changed so it extends the ExtendedRepository

File: .github/workflows/microservices.yml [hunk 2]
```diff
@@ -53,11 +53,11 @@ jobs:
...
 
-    - name: Run API validator
-      run: cd DCSA-API-Validator && mvn test -Dtest.suite=EBLV1.xml
-      env: 
-        client_secret: ${{ secrets.AUTH0_DCSA_CLIENT_SECRET }}
-        client_id: 1qEYXCbxWvjeYS8VVm1BfnfZrP3TY9n8
-        audience: localhost
-        OAuthTokenUri: https://dcsa.eu.auth0.com/oauth/token
-        API_ROOT_URI: http://localhost:9090/v1
+#    - name: Run API validator
```
Comment (pimg) [inline .github/workflows/microservices.yml:56]: shouldn't this be removed

File: src/main/java/org/dcsa/ebl/service/impl/EquipmentServiceImpl.java [hunk 3]
```diff
@@ -27,45 +27,47 @@ public EquipmentRepository getRepository() {
...
-                                equipment.setIsShipperOwned(Boolean.TRUE);
-                                return equipmentRepository.insert(equipment);
-                            })).zipWith(Mono.just(shipmentEquipmentTO));
-                }).doOnNext(tuple -> {
-                    Equipment equipment = tuple.getT1();
-                    ShipmentEquipmentTO shipmentEquipmentTO = tuple.getT2();
-                    EquipmentTO newEquipmentTO = MappingUtil.instanceFrom(equipment, EquipmentTO::new, AbstractEquipment.class);
-                    shipmentEquipmentTO.setEquipment(newEquipmentTO);
-                }).then();
+        return Mono.empty();
```
Comment (pimg) [inline src/main/java/org/dcsa/ebl/service/impl/EquipmentServiceImpl.java:30]: shouldn't this be removed

**Review by pimg — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yaml [hunk 4]
```diff
@@ -45,7 +45,7 @@ spring:
 
 auth0:
```
Comment (douradofrazer) [inline src/main/resources/application.yaml:46]: This can be removed, refer to DCSA-BKG `application.yaml`

**Review by douradofrazer — CHANGES_REQUESTED** [review]

**Review by pimg — APPROVED** [review]

**Review by pimg — APPROVED** [review]

**Pull request merged**